io.github.daid.EmptyEpsilon.yml

```diff
@@ -0,0 +1,70 @@
+id: io.github.daid.EmptyEpsilon
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: EmptyEpsilon-wrapper
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --device=input
```

Comment:
device  input will not work. It is not part of any stable release of flatpak.

Reply:
ah, I didn't catch that, then I'll need --device=all for joystick input